packages/amplify-console-hosting/utils/config-utils.js

```diff
@@ -50,10 +50,6 @@ async function initCurrBackendMeta(context, category, resourceName, type, timeSt
     currMetaContent[category] = {};
   }
 
-  if (!currMetaContent[category][resourceName]) {
-    currMetaContent[category][resourceName] = {};
-  }
-
   currMetaContent[category][resourceName] = metaData;
```

Comment:
are we sure that these paths will be defined? We could use `_.set(currMetaContent, [category, resourceName], metaData` to avoid the extra logic but still get protection against undefined paths

Reply:
This should just be removing redundant assignments. For example, here we were conditionally assigning and then immediately unconditionally assigning `currMetaContent[category][resourceName]`.